src/status_im/contexts/chat/group_details/view.cljs

```diff
@@ -167,4 +168,15 @@
        :render-section-footer-fn       contacts-section-footer
        :render-data                    {:chat-id chat-id
                                         :admin?  admin?}
-       :render-fn                      contact-item-render}]]))
+       :render-fn                      contact-item-render}]
+     [quo/floating-shell-button
+      {:key     :shell
+       :jump-to {:on-press            (fn []
+                                        (rf/dispatch [:navigate-back])
+                                        (debounce/throttle-and-dispatch [:shell/navigate-to-jump-to]
+                                                                        500))
+                 :customization-color profile-color
+                 :label               (i18n/label :t/jump-to)}
+      }
+      style/floating-shell-button
+     ]]))
```

Comment:
- Why `:navigate-back`?
- I think throttle-and-dispatch is not needed
- Please fix brackets formatting

Reply:
I think simple `#(rf/dispatch [:shell/navigate-to-jump-to])` should work